source/includes/get-started/quickstart.c

```diff
@@ -0,0 +1,41 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_cursor_t *results;
+    const bson_t *doc;
+    char *str;
+
+    mongoc_init ();
+
+    client = mongoc_client_new ("<connection string>");
+    collection = mongoc_client_get_collection (client, "sample_mflix", "movies");
+
+    {
```

Comment:
Suggest removing `{` and `}` scope since it does not appear to logically group or seem necessary to make `query` local to this scope.